File: feature1/src/main/java/com/example/andreip/myapplication/feature1/MainActivity.java
package com.example.andreip.myapplication.feature1;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.LinearLayoutManager;
import android.support.v7.widget.RecyclerView;


public class MainActivity extends AppCompatActivity {

    RecyclerView recyclerView;
    String[] myList = {"asta", "este", "viata"};

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        recyclerView = findViewById(R.id.RVRecyclerView);
        recyclerView.setLayoutManager(new LinearLayoutManager(this));
        recyclerView.setAdapter(new MyAdapter(myList));


    }

}

File: feature1/src/main/java/com/example/andreip/myapplication/feature1/MainActivity.kt
package com.example.andreip.myapplication.feature1

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.TextView

class MainActivity : AppCompatActivity() {

    var preffered: String = "aaa";

    fun firstFunction(string: String) : Boolean {
        if (string.equals("aaa"))
            return true;
        return false;
    }

    fun secondFunction(string: String): Boolean {
        val b: Boolean? = null
        if (b ?: false) {
            println(b)
        }
        return false
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var a = 7
        var b = 8
        var calculator: Calculator = Calculator();

        Log.d("Testing", calculator.multiply(a, b).toString())

    }
}
